src/tools/perf/perftest.c

```diff
@@ -194,6 +196,8 @@ ucs_status_t init_test_params(perftest_params_t *params)
     params->super.ucp.send_datatype = UCP_PERF_DATATYPE_CONTIG;
     params->super.ucp.recv_datatype = UCP_PERF_DATATYPE_CONTIG;
     params->super.ucp.am_hdr_size   = 0;
+    params->super.ucp.dmn_local_addr  = empty_addr;
```

Comment:
Minor: I don't think that `empty_addr` is required here and would prefer to use {}.

Reply:
I would also prefer to use `{}` here, and that was my first attempt actually.
But C compiler complains that this syntax is not valid:
```
perftest.c:197:41: error: expected expression before ‘{’ token
     params->super.ucp.dmn_local_addr  = {};
```
So this syntax works for initialization, but not for assignment.
Therefore we have 2 options left: assign temporary struct, or memset. I prefer the first one